pkgs/development/python-modules/django-simple-captcha/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchurl, buildPythonPackage, mock, 
+django, django-ranged-response, pillow }:
+
+buildPythonPackage rec {
+  pname = "django-simple-captcha";
+  version = "0.5.6";
+
+  src = fetchurl {
+    url = "https://files.pythonhosted.org/packages/4d/46/44aff307e370e873ebb44dd8e9a1bcde10ddd1e55779361a5654d273d939/django-simple-captcha-0.5.6.zip";
+    sha256 = "02sbbsjlxa445hdfpxj4wmy1ynfj1vj0xblqhs7r05a2pbl3dzfn";
+  };
+
+  doCheck = false;
```

Comment:
Could you add an explanation why the tests are disabled?

Reply:
There's no much to explain, I just wasn't able to run them :)
Updated the code as per [this](https://github.com/NixOS/nixpkgs/pull/64523#discussion_r301790920) suggestion

Reply:
We should try to make them work, what was the problem?